### DISCUSSION TITLE
Separate title/artist/album columns in "Songs" with adjustable width

### DISCUSSION THREAD

**Discussion by ParaDoX1994** [ciderapp/Cider#1294] [Feature Request]
I'm personally fan of the "Songs" view used in the official Apple Music app on macOS and iTunes because it allows for very condensed list and I'd like to have this UI option in Cider.

How it is currently:

![image](https://user-images.githubusercontent.com/35791009/179973099-0b4907f7-23ac-43a9-88f0-9952d9aee23d.png)

How I'd want it:

![image](https://user-images.githubusercontent.com/35791009/179973608-4bb62390-37e8-4d53-8bd1-b526a79a9960.png)

**Comment by KillerParticles** [comment]
So there is a way to condese the list, its in the rop right corner, this is what it looks like condensed:
![image](https://user-images.githubusercontent.com/93286201/179976689-d54d465c-a1c7-4851-a965-88535aed1ead.png)

If you want it to look more like MacOS or iTunes in regards to the layout of the song title, album and artist, iTheme offers this:
![image](https://user-images.githubusercontent.com/93286201/179976914-86fba98d-a582-4d2b-ba1a-4dec19a20328.png)
![image](https://user-images.githubusercontent.com/93286201/179977047-0d696cac-22e0-4a2f-944e-5b912c07027d.png)

**Reply by ParaDoX1994** [reply]
That's a little better but I feel it's not enough, I just have harder time reading the UI with the artist and album being under the song title instead of next to them, that's mainly what I'm after. In fact is one of the reasons why I ditched Spotify for Apple Music, when they rolled out their new UI design.

The Songs section in macOS is using a similar view as the old iTunes. 

![image](https://user-images.githubusercontent.com/35791009/179977773-678e79f8-9ead-4c95-8701-70c902ca231d.png)

**Comment by KillerParticles** [comment]
Yeah i agree, ill see if i can recreate this in iTheme and ill get back to you as i want it too.

**Reply by ParaDoX1994** [reply]
Appreciate it, thank you.
